src/main/java/seedu/address/storage/exceptions/LoadException.java

```diff
@@ -0,0 +1,14 @@
+package seedu.address.storage.exceptions;
+
+import seedu.address.AlfredException;
+
+/**
+ * Represents an error in loading from JSON file or Storage file into AddressBook when the app is reopened.
+ */
+public class LoadException extends AlfredException {
+    public LoadException(String message, Throwable cause) {
+        super(message, cause);
+    }
+
+}
```

Comment:
Is there an extra closing brace here?

Reply:
What do you mean haha, i couldnt find the extra brace.

Reply:
Same as above, think I commented on the old commit lol 🕐